### PR TITLE
Add new field isRoomClean

### PR DESCRIPTION
**Pull request by bryanhce**
Edits made:

- Prefix rc/
- Add field "isRoomClean" to the Guest Class
- Allow edit of isRoomClean field
- Create a label to display "Room Cleaned: "
- Add test cases to test isRoomClean field
- Minor refactoring of code (person -> guest)

### CODE REVIEW

**Review by Lan-Jingbo — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ guest management tasks done faster than traditional GUI apps.
 
    * **`add`**`n/John Doe p/98765432 e/johnd@example.com dr/13/09/22 - 15/09/22 ng/3`: Adds a guest named `John Doe` to the guest book.
```
Comment (jehousoh) [inline docs/UserGuide.md]: Should we add rc/ here and line 90, 101 and 102?

File: src/test/java/seedu/guest/logic/parser/AddCommandParserTest.java [hunk 2]
```diff
@@ -48,38 +52,38 @@ public void parse_allFieldsPresent_success() {
...
 
         // multiple numbers of guests - last number of guests accepted
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
                 + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_AMY
-                + NUMBER_OF_GUESTS_DESC_BOB, new AddCommand(expectedGuest));
-    }
+                + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB, new AddCommand(expectedGuest));
 
-    @Test
-    public void parse_optionalFieldsMissing_success() {
```
Comment (jehousoh) [inline src/test/java/seedu/guest/logic/parser/AddCommandParserTest.java:77]: Why was this test deleted?

**Review by jehousoh — CHANGES_REQUESTED** [review]
Looks good with some minor changes. Also do remember to merge the latest master before merging.

File: docs/UserGuide.md (hunk 1 above)
Comment (bryanhce) [reply docs/UserGuide.md]: Yes, you are right. I have added them.

File: src/test/java/seedu/guest/logic/parser/AddCommandParserTest.java (hunk 2 above)
Comment (bryanhce) [reply src/test/java/seedu/guest/logic/parser/AddCommandParserTest.java:77]: We no longer have optional fields as such this test is redundant. I have checked with Zong Xian (who removed the Tag field which was optional) and he agreed to remove it.

**Review by jehousoh — APPROVED** [review]
lgtm

**Pull request merged**